packages/config/config/devices/0x010f/fgs214.json

```diff
@@ -35,121 +35,297 @@
 	"paramInformation": [
 		{
 			"#": "1",
-			"label": "Remember relays state",
-			"description": "This parameter determines the state of relays power supply failure (e.g. power outage). auto OFF and flashing modes the parameter not relevant and the relay will always remain off.",
+			"label": "Relays State After Power Failure",
 			"valueSize": 1,
 			"minValue": 0,
 			"maxValue": 2,
 			"defaultValue": 1,
-			"unsigned": true,
 			"allowManualEntry": false,
 			"options": [
 				{
-					"label": "Relays remain switched off after restoring power",
+					"label": "Always off",
 					"value": 0
 				},
 				{
-					"label": "Restore remembered state of relays after restoring power",
+					"label": "Previous state",
 					"value": 1
 				},
 				{
-					"label": "Restore remembered state of relays after restoring power, but for toggle switches (parameter 20/21 set to 1) set the same state as the current state of the switches",
+					"label": "Same as switches",
```

Comment:
I think this needs to mention the behavior when toggle switch (synchronized) is selected. Also "same as switches" is only true when param 20/21 is not set to 1.

Reply:
Something like "Toggle switch (synchronize output): same as switches – other switch types: previous state"? I'm not convinced it's clear enough.

Reply:
Lets do "Toggle switch (synchronize output): current switch state / Other switch types: previous state"

Reply:
Ok, I just pushed the updated version.